packages/blade/src/components/Chip/ChipGroup.tsx

```diff
@@ -97,17 +97,7 @@ const _ChipGroup = (
               <Text>{accessibilityLabel}</Text>
             </VisuallyHidden>
             <BaseBox display="flex" flexDirection="row" flexWrap="wrap">
-              {React.Children.map(children, (child, index) => {
-                return (
-                  <BaseBox
-                    key={index}
-                    marginBottom={chipGroupGapTokens[size].bottom}
-                    marginRight={chipGroupGapTokens[size].right}
-                  >
-                    {child}
-                  </BaseBox>
-                );
-              })}
+              {React.Children.map(children, (child) => child)}
```

Comment:
Again, we need to keep the row gap etc props to make sure we are not changing the existing spacing that was there PLUS we let consumers handle the layout if they want. 

```
            <BaseBox
              display="flex"
              flexDirection="row"
              flexWrap="wrap"
              // Keep existing spacing that was there
              rowGap={chipGroupGapTokens[size].bottom}
              columnGap={chipGroupGapTokens[size].right}
              marginBottom={chipGroupGapTokens[size].bottom}
            >
              {React.Children.map(children, (child, index) => {
                return <BaseBox key={index}>{child}</BaseBox>;
              })}
            </BaseBox>
```

Reply:
https://github.com/razorpay/blade/blob/98296fb7f8da7dfbb9a9db44e5b7e0fa957ddf51/packages/blade/src/components/Chip/Chip.tsx 
Moved that to Chip; consumers can modify it from there.
Adding spacing on the wrapper won't work since this spacing needs to be applied to Chip.

Reply:
@tewarig reusable components should not have margins, they are bit [harmful](https://mxstbr.com/thoughts/margin). If in future we want to expose Chip as a standalone component (eg: Clickable only chip) then it will break component encapsulation. 

> Adding spacing on the wrapper won't work since this spacing needs to be applied to Chip.
Thus the BaseBox i've added on my example.